projects/RPi2/filesystem/usr/lib/systemd/system/serial-console.service

```diff
@@ -1,23 +0,0 @@
-[Unit]
```

Comment:
Don't worry, it will use `projects/RPi/filesystem/usr/lib/systemd/system/serial-console.service`